src/Cello/monitor_Monitor.cpp

```diff
@@ -115,6 +117,7 @@ void Monitor::header () const
   print ("Define","BYPASS_CHARM_MEM_LEAK %s","Yes");
 #else
   print ("Define","BYPASS_CHARM_MEM_LEAK %s","no");
+#endif
 #endif
```

Comment:
```suggestion
#endif
#endif /* THIS_NEEDS_TO_BE_INCLUDED_AGAIN_WHEN_AUTO_CONFIG_IS_POPULATED */
```
minor suggestion for readability